ocaml/xapi/xapi_host.ml

```diff
@@ -885,7 +885,7 @@ let set_ssl_legacy ~__context ~self ~value =
 	let old = Db.Host.get_ssl_legacy ~__context ~self in
 	Db.Host.set_ssl_legacy ~__context ~self ~value;
 	if old <> value then (
```

Comment:
Yes, they are.
The Xapi_mgmt_iface one uses a mutex to protect the restart_stunnel function.